README.rst

```diff
@@ -25,16 +25,22 @@ specified Fileset ID:
 
 ::
 
-    $ omero mkngff sql --symlink_repo /OMERO/ManagedRepository --secret=secret 1234 /path/to/fileset.zarr > myNgff.sql
+    $ omero mkngff sql --symlink_repo /OMERO/ManagedRepository --secret=secret --bfoptions 1234 /path/to/fileset.zarr > myNgff.sql
     $ psql -U omero -d idr -h $DBHOST -f myNgff.sql
 
-To ONLY perform the symlink creation:
+To ONLY perform the symlink creation (and optionally create fileset.zarr.bfoptions)
 
 ::
 
-    $ omero mkngff symlink /OMERO/ManagedRepository 1234 /path/to/fileset.zarr
+    $ omero mkngff symlink /OMERO/ManagedRepository 1234 /path/to/fileset.zarr --bfoptions
```

Comment:
Do you _ever_ run this without "--bfoptions"?

Reply:
Hmmm - good point... probably not! I think we always need the `.bfoptions` file alongside the symlinks since we currently *always* exclude the `chunks` from new Filesets.
So, I guess I'll remove this option from the `symlink` command then...